src/metadataparsers/mods/CdmToMods.php

```diff
@@ -81,21 +68,6 @@ public function __construct($settings)
         }
     }
 
-    private function getMappingsArray($mappingCSVpath, $numOfFields = 3)
-    {
-
-        $filename = $mappingCSVpath;
-
-        $reader = Reader::createFromPath($filename);
-        $collectionMappingArray = array();
-        foreach ($reader as $index => $row) {
-            $collectionMappingArray[$row[0]] = $row;
-
-        }
-
-        return $collectionMappingArray;
-    }
-
```

Comment:
This was exactly the same in `CsvToMods` and `CdmToMods`, so I moved it up to `Mods` and then any new implementations can reuse it or override it.